README.md

```diff
@@ -1,13 +1,34 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
+Some learing goals for this week included:
+- How to use POST requests to send data to your API
+- How to store data in your database from POST requests
+- How to validate data and ensure your database only contains 'good' data
+- How to build a full API which includes handling of user input
+- How to build an API which works well with an existing frontend
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+These are my endpoints used in frontend: 
+
+app.get: /thoughts (will return array of user post)
+<br>
+app.post: /thoughts (for saving and sending data to API)
+<br>
+app.post: /thoughts/:thoughtId/like (find thought by ID, and update its hearts property to add one like.)
+
+Other endpoints in backend but not used in frontend:
```

Comment:
I love the fact that you include the methods you've learned - even though there are not used in your frontend. Makes awareness to viewers of your knowledge.